func_test.go

```diff
@@ -186,41 +162,3 @@ func TestABI(t *testing.T) {
 		}
 	}
 }
-
```

Comment:
I would also keep this here and just modify it to support windows. The reason is it is also used by Linux so if it's pulled out it'd create another file which isn't necessary